packages/data/addon/mirage/fixtures/bard-meta-dimensions.js

```diff
@@ -228,6 +228,14 @@ export default {
       category: 'test',
       datatype: 'text',
       storageStrategy: 'loaded'
+    },
+    {
+      name: 'Budget',
+      longName: 'Budget',
+      cardinality: 462664,
+      category: 'test',
+      datatype: 'number',
+      storageStrategy: 'loaded'
```

Comment:
Will fili set this to `loaded` for datatype `number`?

Reply:
Yup stole it from fili

Reply:
Would it be `storageStrategy: 'none'` since the numbers are loaded into fili cache?

Reply:
no that's backwards. storageStrategy is 'none' when it's NOT loaded into fili cache.

Reply:
Oops, sorry that was a typo, I meant to say:
> Would it be storageStrategy: 'none' since the numbers are NOT loaded into fili cache?

Are these number loaded into the cache?

Reply:
yeah according to the reference implentation. It's giving us 'loaded'. I asked if that is accurate. if it isn't I'll update.